_docs/dev-setup.md

```diff
@@ -91,88 +89,59 @@ For OSX, the installation process will seem rather complicated, but you should e
+What this does is grab the copy of our website located on GitHub, and creates a local copy on our computer.
 
-Almost all computers have ruby installed. To double-check, you can run the following command in your shell:
+To see what link you need to input when cloning a repository, just press the code button on the top of the [repository](https://github.com/uclaacm/teach-la-website) you want to work on and copy the url that appears to your clipboard!
 
-```sh
-$ ruby -v
-ruby 3.0.0p0 (2020-12-25 revision 95aff21468) [x86_64-darwin20]
-```
+![Github Cloning URLs]({{ site.baseurl }}/img/docs/git-url.png)
 
-If you don't have Ruby on your system, you can check the [appendix entry: installing ruby with rvm](#appendix-installing-ruby-with-rvm).
+From here on, we'll use the teach-la-website as our example of navigating git!
 
-First, we'll install a "helper" tool called `bundler` that lets us install libraries easily.
+Once that command is done, a new folder will have been created for your project. We can check if it's there with the `ls` command:
 
 ```sh
-$ gem install bundler
+$ ls
+js-backend       teach-la-website # <-- there it is!
+gradient-descent-visualiser  network-tarines
 ```
 
-If this command doesn't execute properly, then there's two things you can do:
-1. Run `sudo gem install bundler` instead.
-2. Install RVM [using our appendix entry](#appendix-installing-ruby-with-rvm) and then re-run the command.
-
-Then, we'll run a new command, called `bundle`. It will install all of our dependencies.
-
-*Note: this might take a few minutes, depending on your internet connection and computer.*
+We can then enter the directory with the `cd` command, so we can do some more things.
 
 ```sh
-$ bundle
+$ cd teach-la-website
 ```
 
-Great! That's it, and we're ready to start actually making changes!
-
-One last tidbit: every now and then, we'll make changes that update our dependencies. The command line will tell you to run `bundle` again (just like we did), so you should do just that.
-
-## Development Workflow
-
-Okay, so how do we make changes to our site? We'll walk you through what you need to do to get your site up and running.
-
 ### Create a New Branch
 
 We want to make sure that changes we make don't impact other people, or at least not until those changes are done. Out of courtesy, we'll develop our changes on our own "branch", which is a version of our code.
 
-The default branch is called `master`, and it contains the code that goes live on [teachla.uclaacm.com](https://teachla.uclaacm.com). We don't want to touch that just yet, so we're going to make our own branch!
+Default branches are either called `master` or `main`, and they contain the code that goes live on our websites' links! For instance, `teach-la-website`'s master branch code can be seen on [teachla.uclaacm.com](https://teachla.uclaacm.com). We don't want to touch that just yet, so we're going to make our own branch!
```

Comment:
In the future, we should shift all of our branches to `main` - makes writing this a lot easier.